package.json

```diff
@@ -32,7 +32,7 @@
     "prepare": "snyk protect",
     "renderTemplates": "ts-node src/generate.ts",
     "pretest": "npm run lint && npm run depcheck",
-    "test": "nyc mocha \"src/**/*.test.ts\"",
+    "test": "snyk test && nyc mocha \"src/**/*.test.ts\"",
```

Comment:
why are we adding synk test to the test command?

Reply:
The result of `snyk test` will change over time, so test results will not be reproducible. I don't think that we should do this. Also because we already have the same functionality from our Snyk monitoring integrations. (I think. Snyk docs aren't _super_ clear.)

If we do want `snyk test` to be run, it should be as a `postinstall` script, not a `test` script. Like `npm audit` is.